### ISSUE TITLE
add 2 feature to plugin - it's my offer !

### ISSUE THREAD

**Issue by khosrobaigy** [khosrobaigy/2features#1]
Dear Mr. Carson McDonald
I am Khosrobaigy.
I have my own blog
In addition, i am using the  Google Analytics
also , i am Using "cgi counter hit" service from my Webhost services

Previously, there was no statistical problem and  this counter would be compatible with Google Analytics pageview
It was also possible that (if necessary) I set the value for cgi counter

But recently I've found the following problems:
1. the cgi Counter value finds large differences with Google Analytics pageview
2. I can not (have not permission) to initialize the cgi counter. I must get help from support of my hostting service to change the amount of cgi coumter

This troubles force me to have use your plug-in (Google Analytics Dashboard) in my sidebar (to display the pageview of Google Analytics)
I want to make the following changes into your plug-in source:
1. change the Start date of displaying Google Analytics data to date "Aug 10, 2008"
2. I would also like to display "amount of Google pageview" on all pages!
I display the Google pageview (which is now 663,229 - according to Figure 3)
And I do not want the hits of each page individually displayed
In other words, I want the number displayed by your plug-in (Figure 1) on main page
And on other pages (for example fig 2)
to be similarly (same to same) !!!

Could you tell me please?
Instead
$ Link_uri = substr ($ _SERVER ["REQUEST_URI"], - 20);
What phrase (and Where?) I replace
That's my goal !
- Please tell me basically what changes need to be made at the source?

Also, I offer to add this opportunity  on your plug-in (parametric):
1. Setting the start date for displaying pageview hits
2. Display the number of pageviews in all pages (the same as cgi counter)

Tnx
